guides/plugins/plugins/framework/data-handling/entities-via-attributes.md

```diff
@@ -434,11 +589,36 @@ class ExampleEntity extends Entity
     #[ManyToMany(entity: 'currency', onDelete: OnDelete::CASCADE)]
     public ?array $currencies = null;
 
+    /**
+     * @var array<string, OrderEntity>
+     */
+    #[ManyToMany(entity: 'order', onDelete: OnDelete::CASCADE)]
+    public ?array $orders = null;
+
     /**
      * @var array<string, ArrayEntity>|null
      */
     #[Translations]
     public ?array $translations = null;
 }
 
+
+<?php declare(strict_types=1);
+
+namespace Examples;
+
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+
+/**
+ * @internal
```

Comment:
Why would one suggest others, that entity classes are internal. Internal compared to what?

Reply:
thanks for the hint! copy pasta 😅 Will fix it tomorrow.